frontend/src/metabase/query_builder/components/LimitPopover.jsx

```diff
@@ -12,7 +13,7 @@ const CustomRowLimit = ({ limit, onChangeLimit, onClose }) => {
     <LimitInput
       small
       defaultValue={limit}
-      className={cx({ "text-brand border-brand": limit != null })}
+      className={limit != null ? cx(CS.textBrand, CS.borderBrand) : null}
```

Comment:
This might be better as
```
className={ cx({ [cx(CS.textBrand, CS.borderBrand)] : limit != null }) }
```

Reply:
Yeah that looks better. As it looks more close to the previous version.